src/components/app/details/appDetails/SourceInfo.tsx

```diff
@@ -91,16 +92,16 @@ export function SourceInfo({
                         <button
                             className="cta cta-with-img small cancel fs-12 fw-6"
                             onClick={(e) =>
-                                showHibernateModal(status.toLowerCase() === 'hibernating' ? 'resume' : 'hibernate')
+                                showHibernateModal(isHibernated ? 'resume' : 'hibernate')
```

Comment:
Not a new change but can we keep the resume and hibernate constant?

Reply:
As discussed, we'll be covering all messaging/hardcoding to const.